shoreline.go

```diff
@@ -53,7 +53,15 @@ func main() {
 		logLevel = log.WarnLevel
 	}
 	logger.SetLevel(logLevel)
-	logger.Infof("Starting shoreline service %v\n", version.GetVersion().String())
+
+	// Development mode?
+	appEnv := os.Getenv("APP_ENV")
+	if appEnv == "development" {
+		logger.Info("starting shoreline in development mode")
```

Comment:
I feel like this log is not necessary as we have a similar one line 64, or am I missing something?

Reply:
correct...